src/core/Menu/Menu.tsx

```diff
@@ -48,8 +60,8 @@ export const Menu = React.forwardRef<HTMLUListElement, MenuProps>(
       const selectedIndex = items.findIndex(
         (el) => el.getAttribute('aria-selected') === 'true',
       );
-      setFocusedIndex(selectedIndex > -1 ? selectedIndex : 0);
-    }, [focusedIndex]);
+      setFocus && setFocusedIndex(selectedIndex > -1 ? selectedIndex : 0);
```

Comment:
Convert to `if` condition, it is not an inline function or anything.

Reply:
Done